atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/DdlConfig.java

```diff
@@ -61,6 +63,16 @@ public int mutationBatchSizeBytes() {
         return 2 * 1024 * 1024;
     }
 
+    @Value.Default
+    public int parallelDdlOperationConcurrency() {
+        return 16;
```

Comment:
I don't see that as a problem, the actual overall KVS calls are synchronous/blocking so if a user does the sane thing and says
createTables(foo,bar)
and then dropTables(foo,bar)
It will still always be correct.

A user introducing their own concurrency with drop and create tables simultaneously already could have had issues even without the extra parallelism here.